src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -32,11 +32,11 @@ Users who need manual instrumentation should reference the "Datadog.Trace" packa
     <ProjectReference Include="..\Datadog.Trace\Datadog.Trace.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+  <ItemGroup Condition="' $(TargetFramework)' == 'netstandard2.0'">
```

Comment:
Not trying to be super nitpicky, but I'm worried that the introduced space here could make this equality false. Can you double check this?

Reply:
Sure!